--- Jamfile.v2	2009-12-14 13:57:33.000000000 +0100
+++ Jamfile.new	2009-12-14 14:00:29.000000000 +0100
@@ -8,6 +8,8 @@
     : source-location ../src
     : requirements <link>shared:<define>BOOST_TEST_DYN_LINK=1
                    <toolset>msvc:<asynch-exceptions>on
+                   <toolset>gcc:<cxxflags>-Wno-long-long 
+                   <toolset>darwin:<cxxflags>-Wno-long-long  
                    <toolset>borland:<cxxflags>-w-8080
                    # Disable Warning about boost::noncopyable not being exported
                    <link>shared,<toolset>msvc:<cxxflags>-wd4275
@@ -107,4 +109,4 @@
 boost-install boost_prg_exec_monitor 
               boost_test_exec_monitor
               boost_unit_test_framework ;
-          
\ No newline at end of file
+          
